packages/design-tokens/src/tokens/functional/size/size.json

```diff
@@ -79,6 +68,20 @@
         }
       }
     },
+    "borderRadius": {
+      "small": {
+        "value": "{base.size.4}"
+      },
+      "medium": {
+        "value": "{base.size.8}"
+      },
+      "large": {
+        "value": "{base.size.16}"
+      },
+      "xlarge": {
+        "value": "{base.size.24}"
+      }
+    },
```

Comment:
This would conflict with the same tokens shipped in `border.css`. These ones are inherited from Primitives 👇 

![Screenshot 2023-09-21 at 15 38 57](https://github.com/primer/brand/assets/13340707/67b3a980-f124-4909-a48c-2be6d70f9c74)

I think it's fine to move the tokens into size.json, but we'd want to move the [other ones from that file](https://unpkg.com/browse/@primer/brand-primitives@0.26.0/lib/design-tokens/css/tokens/functional/size/border.css) into this one and then remove the border.css file altogether.

Reply:
I just added the missing `--brand-borderRadius-full` to the size.json file (https://github.com/primer/brand/pull/428/commits/fbe86140e587cb2eee6b0c12615900357ed020c6)  and this is the new output of the `border.css`👇 

<img width="747" alt="Screenshot 2023-09-21 at 17 03 45" src="https://github.com/primer/brand/assets/912236/55a7f58f-c0a7-4323-9540-658629b64f51">

not sure how the token build works 😓 , but it seems that it avoids an output with tokens using the same nomenclature, so I can't see how this really conflicts 🤔 

If we still want to remove the border.css file, should that be done in the `build-token.js` file? (i might need some help here lol)

Reply:
👍 your change should avoid the conflict. 

On the topic of where these tokens should reside, I just spoke to @langermank and confirmed that v8 will retain the borders.css output file, which matches the current way we do it in Primer Brand.

For reference, here's the v8 Primitives output: https://unpkg.com/browse/@primer/primitives@7.13.1/tokens-next-private/css/functional/size/border.css

My recommendation would be to pull the border radius tokens out of `size.json` and put them into a new file called `border.json` in `packages/design-tokens/src/tokens/functional/size/`. You'd also want to move over the borderWidth and and borderInset values into that file too. This should mean we're staying consistent and aligned with the direction of travel of Primitives, and make the inevitable upgrade a little bit easier. 

@josepmartins - I'm happy to help with / do this part ☝️ if you're okay with me pushing the change up? lmk.

Reply:
sounds good @rezrah thanks for the explanation! Feel free to push the changes 🙏